system-parachains/collectives/collectives-polkadot/src/secretary/mod.rs

```diff
@@ -0,0 +1,294 @@
+	pub SecretarySalaryInteriorLocation: InteriorLocation = PalletInstance(<crate::SecretarySalary as PalletInfoAccess>::index() as u8).into();
+}
+
+const USDT_UNITS: u128 = 1_000_000;
+
+/// [`PayOverXcm`] setup to pay the Secretary salary on the AssetHub in USDT.
+pub type SecretarySalaryPaymaster = PayOverXcm<
+	SecretarySalaryInteriorLocation,
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	AccountId,
+	(),
+	ConvertToValue<AssetHubUsdt>,
+	AliasesIntoAccountId32<(), AccountId>,
+>;
+
+pub struct SalaryForRank;
+impl GetSalary<u16, AccountId, Balance> for SalaryForRank {
+	fn get_salary(rank: u16, _who: &AccountId) -> Balance {
+		if rank == 1 {
+			1000
+		} else {
+			0
+		}
+	}
+}
+
+impl pallet_salary::Config<SecretarySalaryInstance> for Runtime {
+	type WeightInfo = (); // TODO weights::pallet_salary_secretary_salary::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Paymaster = SecretarySalaryPaymaster;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Paymaster = crate::impls::benchmarks::PayWithEnsure<
+		SecretarySalaryPaymaster,
+		crate::impls::benchmarks::OpenHrmpChannel<ConstU32<1000>>,
+	>;
+	type Members = pallet_ranked_collective::Pallet<Runtime, SecretaryCollectiveInstance>;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Salary = SalaryForRank;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Salary = frame_support::traits::tokens::ConvertRank<
+		crate::impls::benchmarks::RankToSalary<Balances>,
+	>;
+	// 15 days to register for a salary payment.
+	type RegistrationPeriod = ConstU32<{ 15 * DAYS }>;
+	// 15 days to claim the salary payment.
+	type PayoutPeriod = ConstU32<{ 15 * DAYS }>;
+	// Total monthly salary budget.
+	type Budget = ConstU128<{ 10_000 * USDT_UNITS }>;
+}
+
+parameter_types! {
+	pub const SecretaryTreasuryPalletId: PalletId = SECRETARY_TREASURY_PALLET_ID;
+	pub const ProposalBond: Permill = Permill::from_percent(100);
+	pub const Burn: Permill = Permill::from_percent(0);
+	pub const MaxBalance: Balance = Balance::MAX;
+	// The asset's interior location for the paying account. This is the Secretary Treasury
+	// pallet instance.
+	pub SecretaryTreasuryInteriorLocation: InteriorLocation =
+		PalletInstance(<crate::SecretaryTreasury as PalletInfoAccess>::index() as u8).into();
+}
+
+#[cfg(feature = "runtime-benchmarks")]
+parameter_types! {
+	pub const ProposalBondForBenchmark: Permill = Permill::from_percent(5);
+}
+
+/// [`PayOverXcm`] setup to pay the Secretary Treasury.
+pub type SecretaryTreasuryPaymaster = PayOverXcm<
+	SecretaryTreasuryInteriorLocation,
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	VersionedLocation,
+	VersionedLocatableAsset,
+	LocatableAssetConverter,
+	VersionedLocationConverter,
+>;
+
+pub type SecretaryTreasuryInstance = pallet_treasury::Instance3;
+
+impl pallet_treasury::Config<SecretaryTreasuryInstance> for Runtime {
```

Comment:
Do we really need this?

Reply:
I also think we do not need referenda pallet. We better add it when if needed.

Reply:
Then there will be no need for tracks?

Reply:
Yeah probably not right now.

Reply:
This will then need a custom implementation of 
```rust
type Polls: Polling<TallyOf<Self, I>, Votes = Votes, Moment = BlockNumberFor<Self>>;
```

Reply:
no-op impl